src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -1045,3 +1045,24 @@ function dorogovtsev_mendes(n::Integer; seed::Int=-1)
     end
     return g
 end
+
+"""
+    random_orientation_dag(n)
+
+Generate a random oriented acyclical digraph. The function takes in a simple/
+graph as an argument. 
+"""
+function random_orientation_dag(g::SimpleGraph{T}) where T <: Integer
```

Comment:
There could also be an in-place version of this, `random_orientation_dag!` randomly re-orienting the existing edges of a graph

Reply:
It would be type-unstable though since the input here is an undirected graph.

Reply:
Could we do this in a separate PR? with regards to the type unstableness, we could generate a digraph from an existing digraph